test/integration.test.js

```diff
@@ -26,8 +26,9 @@ test("Verify GitHub auth", async (t) => {
 
   const fetch = fetchMock
     .sandbox()
-    .getOnce(`https://api.github.local/repos/${owner}/${repo}`, {
+    .get(`https://api.github.local/repos/${owner}/${repo}`, {
       permissions: { push: true },
+      clone_url: `git+https://othertesturl.com/${owner}/${repo}.git`,
```

Comment:
Use `repeat` argument to be explicit:
https://www.wheresrhys.co.uk/fetch-mock/api/mocking/parameters/options/#repeat